app/_hub/kong-inc/acme/_index.md

```diff
@@ -2,7 +2,6 @@
 name: ACME
 publisher: Kong Inc.
 version: 0.4.x
-source_url: 'https://github.com/Kong/kong-plugin-acme'
```

Comment:
Removed this for two reasons: 
* The plugin is no longer in a separate repo, it's now part of the main codebase
* `source_url` keeps triggering a check that is inaccurate (will fix in a future change)